exporters/FusionExporter/Source/AddIn/EUI-Handlers.cpp

```diff
@@ -58,13 +58,15 @@ bool EUI::clearHandler<WorkspaceDeactivatedHandler>(Ptr<UserInterface> UI, Works
 template<>
 bool EUI::addHandler<ShowPaletteCommandCreatedHandler>(Ptr<CommandDefinition> commandDef, ShowPaletteCommandCreatedHandler* showPaletteCommandCreatedHandler)
 {
-	showPaletteCommandCreatedHandler = new ShowPaletteCommandCreatedHandler(this, commandDef->id());
+	ShowPaletteCommandCreatedHandler* showPaletteCommandCreatedHandler_;
+	showPaletteCommandCreatedHandler_ = new ShowPaletteCommandCreatedHandler(this, commandDef->id());
```

Comment:
Please revert changes to this file, the handler object should be passed out by pointer so the handler can be removed later.